### PR TITLE
promote include_subgraph_errors plugin

### PR DESCRIPTION
**Pull request by garypen**
It has been experimental for a while and is now official.

fixes: #1773

### CODE REVIEW

**Review by abernix — APPROVED** [review]
LGTM! one comment on CL.

File: docs/source/configuration/subgraph-error-inclusion.mdx [hunk 1]
```diff
@@ -2,8 +2,6 @@
 title: Subgraph error inclusion
 ---
 
-> ⚠️ Apollo Router support for subgraph error inclusion is currently experimental.
-
```
Comment (abernix) [inline docs/source/configuration/subgraph-error-inclusion.mdx:6]: thank you, btw, for catching this!

File: NEXT_CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,24 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 
 ## ❗ BREAKING ❗
 
+### Promote include_subgraph_errors from experimental ([Issue #1773](https://github.com/apollographql/router/issues/1773))
+
+The include_subraph_errors plugin is promoted from experimental. Configuration changes are required. For example:
+
```
[the remark this answers is not in the document]
Comment (garypen) [reply NEXT_CHANGELOG.md:33]: I don't mind either way.

**Review by o0Ignition0o — APPROVED** [review]

File: NEXT_CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (abernix) [reply NEXT_CHANGELOG.md:33]: let's do it them.

[the remark this answers is not in the document]
Comment (o0Ignition0o) [reply NEXT_CHANGELOG.md:33]: I read it as a mark of irony 😅

[the remark this answers is not in the document]
Comment (abernix) [reply NEXT_CHANGELOG.md:33]: ah, nvm.  i'll add the changes on the release PR.

**Comment by BrynCooke** [comment]
I was too late to comment on this. I'm not massively keen on the name of the plugin, and feel that eventually it could be part of a larger dev plugin in the config. I guess if and when that happens we can just deprecate this one.

**Comment by abernix** [comment]
@BrynCooke Let's definitely chat about it.  I agree this will need to evolve, though many production users have _actually_ taken the effort to make their subgraph errors meaningful and safe and want them exposed, so they'll want a way to do that.

File: apollo-router/src/configuration/mod.rs [hunk 3]
```diff
@@ -189,7 +189,7 @@ impl Configuration {
             Value::Bool(true),
         );
         self.plugins.plugins.as_mut().unwrap().insert(
-            "experimental.include_subgraph_errors".to_string(),
+            "apollo.include_subgraph_errors".to_string(),
```
Comment (bnjjj) [inline apollo-router/src/configuration/mod.rs:192]: Hmmm I think now it's an apollo one, it's no longer located in `self.plugins` but in `self.apollo_plugins` isn't it ?

Comment (garypen) [reply apollo-router/src/configuration/mod.rs:192]: Ok. Re-reading. I think this was already in the wrong place. `experimental` plugins should be in `apollo_plugins`. Anyway, it definitely needs to be fixed.

Comment (garypen) [reply apollo-router/src/configuration/mod.rs:192]: I think it is in the wrong place, but the processing of mandatory plugins positions it correctly. We should still fix this at some point.

Comment (garypen) [reply apollo-router/src/configuration/mod.rs:192]: see: #1777

**Pull request merged**